apps-rendering/package.json

```diff
@@ -91,7 +90,6 @@
 		"eslint-plugin-eslint-comments": "^3.2.0",
 		"eslint-plugin-import": "^2.26.0",
 		"eslint-plugin-jsx-a11y": "^6.7.1",
-		"eslint-plugin-prettier": "^4.2.1",
```

Comment:
this wasn't actually being used afaict

Reply:
Indeed, it seems like we use [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier#installation)